### DISCUSSION TITLE
ReactDOM.render is no longer supported in React 18

### DISCUSSION THREAD

**Discussion by sonesay** [grid-js/gridjs#1088] [General]
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more:

Can we update the examples in the documentation so we no longer get this warning please?

**Comment by afshinm** [comment]
@sonesay would you be able to share your code or the stacktrace please? I'm not entirely sure how/when you get that warning.

**Comment by afshinm** [comment]
Just published a new version of `gridjs-react` which should fix this error. Please install the latest version (5.1.0).

**Reply by fpaz** [reply]
@afshinm It's happening again on `6.0.1`

**Reply by fpaz** [reply]
I have submitted a PR https://github.com/grid-js/gridjs-react/pull/414
